prow/cluster/monitoring/prow_alertmanager.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Alertmanager
+metadata:
+  name: prow
+  namespace: prow-monitoring
+spec:
+  replicas: 3
+  baseImage: docker.io/prom/alertmanager
+  listenLocal: false
+  nodeSelector: {}
+  secrets:
+  securityContext: {}
+  serviceAccountName: alertmanager
+  version: v0.15.2
```

Comment:
We might need this part too.
@stevekuznetsov If i recall, you have a docker command to check the USER in docker file ... `docker inspect`?

https://github.com/kubernetes/test-infra/blob/3ce030d42897b8957c273eb93c17a66d8c72862c/prow/cluster/monitoring/prow_prometheus.yaml#L40-L43

Reply:
Should be OK without `securityContext` since the following command returned nothing.

```
# podman inspect docker.io/prom/alertmanager:v0.15.2 | grep -i nobody
```

Reply:
Lets use it so we have similar config.

Reply:
Done.